regexes/oss.yml

```diff
@@ -5,6 +5,24 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+##########
+# KreaTV
+##########
+- regex: 'KreaTV/0.0.0.0'
+  name: 'KreaTV'
+  version: ''
+
+- regex: 'KreaTV/(\d+[\.\d]+)'
+  name: 'KreaTV'
+  version: '$1'
+
+##########
+# Opera TV (https://blogs.opera.com/news/2016/01/ces-2016-meet-opera-tv/)
+##########
+- regex: 'Linux (?:[^;]+); Opera TV(?: Store)?/|^Opera/\d+\.\d+ \(Linux mips'
+  name: 'Opera TV'
+  version: ''
+
```

Comment:
@sgiehl I want your opinion about this stuff?
I noticed that all TVs have UA
`^Opera/\d+\.\d+\(Linux mips` can we mark OS as Opera TV?

Reply:
find all tests by `\( linux mips`
![image](https://user-images.githubusercontent.com/1337066/154091615-4d92cc0d-1b79-4414-9b76-420e1ad062f4.png)

result only findes tv's + opera

Reply:
Then lets go with that for now. If at any time a useragent pops up that isn't detected correctly, we can always consider to change that again...

Reply:
merge?